packages/gatsby-theme-basic/src/components/modularity-modules/TextModule.js

```diff
@@ -32,11 +32,7 @@ export default function TextModule({
   module: {
     content,
     contentMedia,
-    modTextOptions: {
-      // fontSize = "text-md",
-      hideBoxFrame = false,
-      theme,
-    } = {},
+    modTextOptions: { fontSize = "text-md", hideBoxFrame = false, theme } = {},
```

Comment:
We shouldn’t implement the `fontSize` option since it will be removed in Modularity 3.0

Reply:
Ok!